alyssa_mckee/class_assignments/product.py

```diff
@@ -0,0 +1,38 @@
+class Product(object):
+	def __init__(self, price, item_name, weight, brand, cost, status="for sale"):
+		self.price = price
+		self.name = item_name
+		self.weight = weight
+		self.brand = brand
+		self.cost = cost
+		self.status = status
+	
+	def sell(self):
+		self.status = "sold"
+		return self
+	
+	def add_tax(self, tax):
+		return format(self.price + self.price*tax, '.2f')
+	
+	def Return(self, reason):
+		if reason.lower().find("defective") > -1:
```

Comment:
Instead of calling .lower() in each if statement. It seems we could save some processing time by doing this once at the top and then passing the lowercased version of the string around. reason = reason.lower()